wiki/en/Unregistered-Servers.md

```diff
@@ -29,7 +29,7 @@ You should connect yourself using the local network address of the computer the
 
 ### Dynamic DNS and why you will probably need it
 
-Unless your ISP provides you with a fixed IP address, you may find that your address changes over time. You might want to set up "dynamic DNS" to get a static address you can share with others. A "dynamic DNS" provider can supply you with a static IP address that you can share, but you need to keep refreshing your dynamic IP with the provider. Your router might support certain providers to do this. If this is not the case, the provider will have instructions on how to set up a dynamic DNS client.
+Unless your ISP provides you with a fixed IP address, you may find that your address changes over time. You might want to set up "dynamic DNS" to get a static address you can share with others. A "dynamic DNS" provider can supply you with a domain name that you can share, but you need to keep refreshing your dynamic IP with the provider. Your router might support certain providers to do this. If this is not the case, the provider will have instructions on how to set up a dynamic DNS client.
```

Comment:
```suggestion
Unless your ISP provides you with a fixed IP address, you may find that your address changes over time. You might want to set up "dynamic DNS" to get a fixed address you can share with others. A "dynamic DNS" provider can supply you with a domain name that you can share, but you need to keep refreshing your dynamic IP with the provider. Your router might support certain providers to do this. If this is not the case, the provider will have instructions on how to set up a dynamic DNS client.
```

Reply:
As I said, it's redundant usage.

Reply:
Use the **Add a suggestion** interface.

Reply:
It doesn't need a change to what's in the commit.